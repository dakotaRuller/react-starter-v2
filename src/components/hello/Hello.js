//Node Modules
import React from 'react'

const Hello = ({name}) => {
  return (
    <p>Hello {name}</p>
  )
}

export default Hello;
