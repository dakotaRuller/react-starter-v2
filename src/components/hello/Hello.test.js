import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Hello from './Hello';

const name = "Lorem";
describe("<Hello/>" , () => {
  let wrapper;
  let props;
  beforeEach(() => {
    props = {
      name
    };
    wrapper = shallow(<Hello {...props}/>)
  });
  test("Component matches snapshot", () => {
    expect(wrapper.find("p").text()).toEqual(`Hello ${name}`);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
