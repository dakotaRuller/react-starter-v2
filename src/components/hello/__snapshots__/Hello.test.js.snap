// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hello/> Component matches snapshot 1`] = `
<p>
  Hello 
  Lorem
</p>
`;
